app/pom.xml

```diff
@@ -15,6 +15,14 @@
     <name>apicurio-registry-app</name>
     
     <dependencies>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
Not sure whether this is a problem for Quarkus.  Since it's a build-time thing maybe not.  But not sure.